src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import java.io.IOException;
```

Comment:
I appreciate the way that you have ordered your import statements, they are easy to comprehend and stick to the standard's specification about grouping:)